_rules/element-lang-valid-de46e4.md

```diff
@@ -25,22 +25,21 @@ acknowledgments:
 
 ## Applicability
 
-This rules applies to any HTML element that:
+This rules applies to any HTML element with a `lang` [attribute value][] that is not empty (`""`) and for which all of the following is true:
 
-- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html`; and
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) of a `body` element; and
-- has a `lang` attribute that is not empty ("").
+- the element is a [descendant][] in the [flat tree][] of a `body` element; and
+- the element has a [node document][] with a [content type][] of `text/html`.
+- the element has a [visible][] [text node][] as a [descendant][] in the [flat tree][].
 
 ## Expectation
 
 For each test target, the value of the `lang` attribute is a [valid language tag][].
 
 ## Assumptions
 
-- The `lang` attribute is assumed to be used to indicate the language of a section of the content. If the `lang` attribute is used for something else (for example to indicate a `code` element contains CSS), the content may still conform to WCAG despite failing this rule.
```

Comment:
Why was this removed?

Reply:
+1

Reply:
One of the previous comments was citing that this is irrelevant - https://github.com/act-rules/act-rules.github.io/pull/1383#discussion_r452141399, hence removed it.

Let me know if I should add it back.

Reply:
I think this should be put back.